src/platform/user/profile/vap-svc/components/base/VAPServiceEditModal.jsx

```diff
@@ -64,7 +64,6 @@ export default class VAPServiceEditModal extends React.Component {
         isEmpty,
         onCancel,
         title,
-        clearErrors,
```

Comment:
can you get rid of line 17 above as well?

Reply:
@erikphansen I didn't take it off because I left Line 47 to use the function.